intel_extension_for_transformers/llm/runtime/graph/models/model_utils/model_types.h

```diff
@@ -137,6 +152,9 @@ struct model_layer {
 
   // ff
   struct ne_tensor* ffn[MODEL_MAX_FFN];
+
+  struct ne_tensor* k_cache;
+  struct ne_tensor* v_cache;
```

Comment:
Does it duplicated with `model_struct::kv_self`?

Reply:
It works for ChatGLM-2. I will double check these parameters later for the ChatGLM-1 enabling.

Reply:
Not duplicated. The normal KV cache for all layers are stacked in a tensor. But chatglm-2 is that each layer has its own the KV cache.